include/mbedtls/x509_crt.h

```diff
@@ -331,12 +331,22 @@ typedef void mbedtls_x509_crt_restart_ctx;
 /**
  * Default security profile. Should provide a good balance between security
  * and compatibility with current deployments.
+ *
+ * This profile permits:
+ * - SHA2 hashes with at least 256 bits: SHA-256, SHA-384, SHA-512.
```

Comment:
This excludes SHA-224. With 3-DES on its way out, I don't expect SHA-224 to gain in popularity.